__tests__/__e2e__/cli.e2e.js

```diff
@@ -26,7 +26,7 @@ describe('End-to-End CLI', () => {
   })
 
   test('CLI should show vulnerabilities breakdown numbers and their titles', async () => {
-    expect.assertions(7)
+    expect.assertions(8)
```

Comment:
@ohaddahan are you saying I can't count??? 😂

Reply:
I didn't say that, `jest` on the other hand was claiming differently 😆

Reply:
of course! was just kidding :)

Reply:
Same here 👋